packages/suite/src/components/suite/modals/TransactionDetail/components/BasicDetails.tsx

```diff
@@ -16,6 +16,7 @@ import {
     getFeeUnits,
 } from '@suite-common/wallet-utils';
 import { TransactionHeader } from '@suite/components/wallet/TransactionItem/components/TransactionHeader';
+import { fromWei } from 'web3-utils';
```

Comment:
Suite is officially web3 tech from now on 🕸️🕸️🕸️

Reply:
On a serious note, why not use the existing converter? It does the same thing, no?

Reply:
It is in fact exactly the same web3 util as it was before, I've just moved it closer to the consumer as everything else. https://github.com/trezor/trezor-suite/blob/5c385c1d769a2ea07dfe14eb2390436807030112/suite-common/wallet-utils/src/transactionUtils.ts#L594

I don't know if it could be replaced by our converter or not and it wasn't the goal of this PR.

Reply:
agree with @marekrjpolak this way it is much much easier to do PR review. 
Maybe you could write this idea for further refactoring into followups section.